data/org.gnome.Shell.Extensions.GSConnect.gschema.xml

```diff
@@ -182,6 +182,8 @@
     <key name="talking-pause" type="b">
       <default>true</default>
     </key>
+    <key name="after-unpause" type="b">
```

Comment:
I don't love the name here, but I tried to follow the "state-action" format of the other names, like "ringing-pause" and "talking-volume". (Maybe "end-unpause" would be a better choice?) Using "unpause" instead of "resume" was definitely a deliberate choice, to more directly indicate what exactly this controls.

Reply:
I think "call ended" is probably the most recognizable term. "Call Ended" doesn't really flow from the other titles, but hey.

For the GSettings key I'd probably for `ended-pause`. Even though that's sort of inverted, the key itself isn't user-visible and flows from `talking-pause`, etc. I expect someone will ask for something to the effect of `ended-volume` eventually.

That's my two cents :)

Reply:
We've already had a request (https://github.com/GSConnect/gnome-shell-extension-gsconnect/issues/1316#issuecomment-1083714934) for `ended-delay`, which I quickly erected a Somebody Else's Problem Field around.

Reply:
> I think "call ended" is probably the most recognizable term. "Call Ended" doesn't really flow from the other titles, but hey.

So, "Call Ended" for the UI string (the title of the new one-item list that is sure to grow longer eventually), `ened-pause` for the key? Works for me!

Hmmm. Tho... so, `ended-pause` should be _`false`_ by default, then? And the preference would flip it to `true`, if it should not auto-unpause? (Ow, negatives! :exploding_head: ) Man, that really is kind of confusing. But, that's what comments are for, I guess!

Reply:
> And the preference would flip it to true, if it should not auto-unpause?

That would also make the key value be the _inverse_ of the UI control. Ouch.

Maybe `ended-unpause` really is a better idea.

Reply:
`ended-resume` is fine, I was just thinking `foobar-pause` would refer to the paused state rather than "Pause" the action. I'm not into bikeshedding that particular one though :)

Reply:
@andyholmes \*nod\* `ended-pause` would still have the same problem, though, whether it's verb or adjective. Normally we _don't_ want to leave the players in a paused state after the call, so `ended-pause` would default to `false`. But then the UI switch still ends up being inverted, unless you word it something convoluted like, "Pause state continues after call end", or "Keep players paused at end of call".

But whenever I try to come up with a normal-person-speak explanation for what the switch in preferences does, it always contains some variation on "resume" and would be enabled by default. Which means its `true`/`false` states are the inverse of the most-obvious meaning of `ended-pause`.

It's fine. `ended-resume` seems perfectly workable, no reason not to just go with that.